lms/djangoapps/certificates/api.py

```diff
@@ -55,15 +56,16 @@
 
 def _format_certificate_for_user(username, cert):
     """
-    Helper function to serialize an user certificate.
+    Helper function to serialize a user certificate.
 
     Arguments:
         username (unicode): The identifier of the user.
         cert (GeneratedCertificate): a user certificate
 
     Returns: dict
     """
-    try:
+    course_overview = get_course_overview_or_none(cert.course_id)
+    if cert.download_url or course_overview:
```

Comment:
Including `cert.download_url` in the condition allows us to continue to include PDF certificates, which have historically been included (and formatted) through this function.